R/Task.R

```diff
@@ -127,6 +136,17 @@ makeTask = function(type, data, weights = NULL, blocking = NULL, fixup.data = "w
     }
   }
 
+  if (spatial == TRUE) {
+    # check if coords are named 'x' and 'y'
+    if (!any(colnames(data) == "x" | colnames(data) == "y")) {
+      stop("Please rename coordinates in data to 'x' and 'y'.")
```

Comment:
This is going to pass when there's only one of "x" and "y".

Reply:
https://github.com/mlr-org/mlr/pull/1891/commits/6be14bc85da9be2b1f9a5b06a109d5449109ebd2